glances/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  glances:
+    image: ${GLANCES_IMAGE}
+    restart: unless-stopped
+    pid: host
+    privileged: true
+    network_mode: "host"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
```

Comment:
Is docker required for all functionality?

Reply:
I think Glances only uses docker socket access (read-only) to report metrics of containers. My personal use-case is for servers, not containers, so I could remove docker socket access. Or we could variablize it and leave it up to the user?

Reply:
Unfortunately the `ro` is completely and total LIE. This gives read and write privileges.

Reply:
In the case of homepage we made it optional, because not everything in homepage requires it. So I would prefer that if there are still usable features of glances that work without it.

Reply:
If we decide to keep Glances in d.ry, should we just eliminate the docker socker access?

Reply:
Fixed in [e86bce8](https://github.com/EnigmaCurry/d.rymcg.tech/pull/284/commits/e86bce8c4914bb96cc0a5d4f9d7427fc8c3c1ac5)